daliuge-engine/test/manager/test_dim.py

```diff
@@ -393,12 +393,16 @@ def test_fullRound(self):
             # The UID of the two leaf nodes of this complex.js graph are T and S
             # Since the original complexGraph doesn't have node information
             # we need to add it manually before submitting -- otherwise it will
+            import importlib.resources
+            ref = importlib.resources.files('test').joinpath('graphs/complex.js')
+            with ref.open('rb') as fp:
+                complexGraphSpec = json.load(codecs.getreader("utf-8")(fp))
+            logger.debug(f"Loaded graph: {fp}")
             # get rejected by the DIM.
```

Comment:
If the importlib stuff is working as it should, we can remove this commented section.

Reply:
Oops, thank you for catching that.